state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -962,6 +964,19 @@ pub mod pallet {
 				_ => Err(Error::<T>::UnsupportedCall)?,
 			}
 		}
+
+		/// Sets the maximum percentage increase (in number of ticks) of the price of the brought
+		/// asset during a swap.
```

Comment:
nit: I think this merits a better explanation. If it's measured in ticks, it's not really a percentage. It's also not necessarily an increase. Also, I think the price could actually move by more than the maximum expressed by this number (because of the whole max/min thing).

Reply:
Ticks have a direct relationship to a percentage (1 to 1 mapping). It is a percentage measured in ticks.

Not sure what you mean by it is not always an increase?

I'll add clarification regarding the fuzzy nature of the maximum.

Reply:
I thought ticks were multiplicative? They can be mapped to a percentage, but they aren't a representation of a percentage (not like pips/bips for example). 

Re. increase - I meant the price can also decrease but I see you mention it's in terms of the 'bought asset'.